doc/whats_new/v1.1.rst

```diff
@@ -12,6 +12,17 @@ Version 1.1.1
 Changelog
 ---------
 
+- |Enhancement| The error message is improved when importing
```

Comment:
I move them to 1.1.1 because they are not part of 1.1.0
Even if they are categorized as "Enhancement", they are more documentation and error message fixing.

I think this is fine to put them in the bugfix release.

Reply:
Well since we already advertised them in the changelog I think it's ok to include them in the bug fix release.